CHANGELOG.rst

```diff
@@ -3,33 +3,48 @@
 0.20.0
 ------
 
-This release includes improvements to the UX on the UI, internal changes to the Knowledge Graph, a bugfix on the core service
-and support for running sessions in a separate namespace.
+Renku ``0.20.0`` tidies up the RenkuLab session start sequence, as well as an assortment of improvements and big fixes. 
```

Comment:
Is `big fixes` a typo for `bug fixes` or is it intended? "_Big_ fix" sounds odd :grin:

Reply:
Oops!! 😆